go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/gravitational/teleport
 
 go 1.20
 
+toolchain go1.21.0
```

Comment:
If we've already specified the 1.21 toolchain why not bump the `go 1.20` to 1.21 too? This go.mod doesn't affect our API users.

Reply:
I would wait for testplan and for 1.21 to be available in release branches first, but I see this ship alreadt sailed.